idmanager/idmanager.go

```diff
@@ -13,6 +13,8 @@ import (
 	"github.com/Scalingo/sand/store"
 )
 
+var ErrNoIDAvailable = errors.New("no new ID available")
```

Comment:
suggestion: with what I wrote below, suggestion
```suggestion
var ErrNoVNIAvailable = errors.New("no new VNI available")
```

Reply:
I'm not sure I agree that we should talk about "ID" everywhere; I would rather change to talk about VNI everywhere. But that's up to you